src/client/components/voyage/StartVoyage.js

```diff
@@ -0,0 +1,239 @@
+import React, { Component } from "react";
+import Title from "./Title";
+import DateTime from "./DateTime";
+import InputField from "./InputField";
+import Dropdown from "./Dropdown";
+import Checkbox from "./Checkbox";
+
+class StartVoyage extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
You can use class fields to set your state like this: 
```suggestion
    state = { ... }
```
Without having to make a constructor.

Reply:
Comment incorporated.